modules/seqexec/web/server/src/main/scala/seqexec/web/server/http4s/WebServerLauncher.scala

```diff
@@ -180,7 +180,8 @@ object WebServerLauncher extends StreamApp[IO] with LogInitialization with Seqex
         c     <- config
         site  <- IO.pure(c.require[Site]("seqexec-engine.site"))
         giapi <- SeqexecEngine.giapiConnection.run(c)
-        seqc  <- SeqexecEngine.seqexecConfiguration(giapi).run(c)
+        // TODO: Pretty sure thhat this is wrong.
+        seqc  <- SeqexecEngine.seqexecConfiguration(giapi, giapi).run(c)
```

Comment:
@cquiroz Is this right ?

Reply:
it is wrong. this implies the gmp of GPI and GHOST are at the same host.
We need to have a `giapiGpi` and `giapiGhost`